dracut/30ignition/coreos-teardown-initramfs-network.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+# -*- mode: shell-script; indent-tabs-mode: nil; sh-basic-offset: 4; -*-
+# ex: ts=8 sw=4 sts=4 et filetype=sh
+
+set -euo pipefail
+
+
+# Propagate initramfs networking if desired. The policy here is:
+#
+#    - If a networking configuration was provided before this point
+#      (most likely via Ignition) and exists in the real root then
+#      we do nothing and don't propagate any initramfs networking.
+#    - If a user did not provide any networking configuration
+#      then we'll propagate the initramfs networking configuration
+#      into the real root.
+#
+# See https://github.com/coreos/fedora-coreos-tracker/issues/394#issuecomment-599721173
+propagate_initramfs_networking() {
+    if [ -n "$(ls -A /sysroot/etc/NetworkManager/system-connections/)" ]; then
+        echo "info: networking config is defined in the real root"
```

Comment:
Perhaps `s/echo "info"/info "/` from the Dracut lib?

Reply:
updated to use the `info()` fucntion from dracut libs

Reply:
actually, I'm going to revert this change because it appears dracut lib isn't tolerant to `set -u`. I get a lot of errors like:

```
coreos-teardown-initramfs-network[757]: /lib/dracut-lib.sh: line 486: DRACUT_QUIET: unbound variable
```

Reply:
ACK. The dracut-ism is to usually not use `set -u`. However using `set -u` is a pattern in our own code. So LGTM.